i_nav_aws_lambda/src/main/java/i_nav/Location.java

```diff
@@ -30,8 +30,87 @@
 	private double latitude;
 	private double longitude;
 	private boolean active;
-	
-	
+
+	public static JSONArray updateLocation(JSONObject updateLoc){
+		JSONArray JSONArr = new JSONArray();
+
+		String update = "UPDATE `locations`";
+		String set = " SET `short_name` = `?`, `long_name` = `?`, `description` = `?`, `min_x_coordinate` = `?`, `min_y_coordinate` = `?`, `max_x_coordinate` = `?`, `max_y_coordinate` = `?`, `active` = `?`";
```

Comment:
i don't think you need the quotes on the parameters
SET `short_name` = ?, `long_name` = ?